src/config.rs

```diff
@@ -0,0 +1,40 @@
+use crate::utils;
+use std::path::Path;
+use toml::Value;
+
+/**
+ * Extract the [build.target-dir] string from cargo config file
+ * 
+ * - Receive the workspace root path
+ */
+pub fn get_targer_dir_name(path: &Path) -> String {
+    // Check if the file exists
+    if let Some(config_file_path) = utils::find_cargo_config(&path).unwrap() {
+        // Get file content as string
+        let contents = std::fs::read_to_string(config_file_path).unwrap();
+        // Parse file content to toml::Value
+        let config = contents.parse::<Value>();
+
+        // Check if parsed sucessfully
+        if config.is_ok() {
+            let config = config.ok();
```

Comment:
We should probably not allow invalid `.cargo/config.toml` to pass through: can we return a `Result` and use `?` here?

Reply:
That way if an error occurs it uses the default value.

Reply:
An error in parsing means the configuration is _invalid_, and can hence not be trusted: we don't know if the user wants or does not want to set a target folder; that way we enforce they don't write invalid toml.